Prog/Lgunit- ACS Table 1B.sas

```diff
@@ -0,0 +1,212 @@
+/************************************************************************** 
+Program:  LargeUnits_ACS Table 1B.sas 
+Library:  
+Project:  NeighborhoodInfo DC 
+Author:   Yipeng Su 
+Created:  6/20/18
+Version:  SAS 9.4 
+Environment:  Windows with SAS/Connect  
+Description: Create calculate percent affordability of large rental units and then calculate concentration by tract
+Modifications: 
+**************************************************************************/ 
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+%DCData_lib( ACS );
+%DCData_lib( Police );
+%DCData_lib( Vital );
+%DCData_lib( Realprop );
+%DCData_lib( HUD );
+
+%let _years = 2012_16;
+
+data calculate_pct;
+     set ACS.acs_2012_16_dc_sum_tr_tr10;
+	 keep geo2010 numrenterhsgunits_&_years. numownocchu3plusbd_&_years. numrentocchu3bd_&_years.
+          numrtohu3b500to749_&_years. numrtohu3b750to999_&_years.  numrtohu3bunder500_&_years. numrtohu3b1500plus_&_years.
+          pct3baffordable1000 pct3baffordable1500 aff1000median aff1000threequarter aff1500median aff1500threequarter;
+
+          pct3baffordable1500= (numrentocchu3bd_&_years.-numrtohu3b1500plus_&_years.)/numrentocchu3bd_&_years.;
+          pct3baffordable1000= sum(numrtohu3b500to749_&_years.,numrtohu3b750to999_&_years., numrtohu3bunder500_&_years.)/numrentocchu3bd_&_years.;
+     if   pct3baffordable1000 >=0.5 then aff1000median=1;
```

Comment:
Yipeng - these need to be based on the median and 75th percentile  for the indicator across tracts not 50% and 75%. You can run a proc univariate to get those values.